PanCardView/Common/Controls/CircleBorder.cs

```diff
@@ -1,22 +1,21 @@
 ﻿using System.ComponentModel;
+using Microsoft.Maui.Controls.Shapes;
 using PanCardView.Extensions;
-using static System.Math;
 
 namespace PanCardView.Controls
 {
-    public class CircleFrame : Frame
+    public class CircleBorder : Border
     {
-        public static readonly BindableProperty SizeProperty = BindableProperty.Create(nameof(Size), typeof(double), typeof(CircleFrame), 10.0, propertyChanged: (bindable, oldValue, newValue) =>
+        public static readonly BindableProperty SizeProperty = BindableProperty.Create(nameof(Size), typeof(double), typeof(CircleBorder), 25.0, propertyChanged: (bindable, oldValue, newValue) =>
```

Comment:
Default size is now 25, which looks nice across platforms. This is because the old value, 10, was never actually respected on most platforms due to the fact that **Frame** has a default padding of 20. Making the effective minimum size 20.